lib/interfaces/include/IOExpander.h

```diff
@@ -0,0 +1,25 @@
+#ifndef IOEXPANDER_H
+#define IOEXPANDER_H
+
+#include <etl/singleton.h>
+#include "MCP23017.h"
+
+class IOExpander
+{
+public:
+    IOExpander();
+
+private:
+    MCP23017 mcp;
+    uint16_t data;
+    
+public:
+    /* Get/update watchdog state */
```

Comment:
I don't think there's a watchdog on the IOExpander. Wouldn't really make sense for there to be.